lib/activeadmin_addons/active_admin_config.rb

```diff
@@ -1,8 +1,8 @@
 class ActiveAdmin::Views::Pages::Base
-  alias_method :original_add_classes_to_body, :add_classes_to_body
+  alias_method :original_build, :build
 
-  def add_classes_to_body
-    original_add_classes_to_body
-    @body.set_attribute "data-default-select", ActiveadminAddons.default_select
+  def build(*args)
```

Comment:
Unused method argument - args. If it's necessary, use _ or _args as an argument name to indicate that it won't be used. You can also write as build(*) if you want the method to accept any arguments but don't care about them.

Reply:
I'm just mirroring the code from AA. If this needs to be changed, someone ping me.

Reply:
Derp. I should pass the args to the original method.